### DISCUSSION TITLE
Disentangling failed

### DISCUSSION THREAD

**Discussion by leogg-github** [Kinggerm/GetOrganelle#163] [Q&A]
Hi,

I have seen a couple of threads with a similar problem, but I fail to see what is the problem... For a result that gave me no warnings the graph seems to still show some connections with mitochondrial segments, while the one that says "disentangling failed" seems more complex but I still do not clearly see the difference from the two graphs in terms of why the second one failed. Is there a way to know what nodes were selected to build the assembly from the graph?

[extended_K115.assembly_graph.fastg.extend-embplant_pt-embplant_mt.csv](https://github.com/Kinggerm/GetOrganelle/files/8515354/extended_K115.assembly_graph.fastg.extend-embplant_pt-embplant_mt.csv)
[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/8515355/get_org.log.txt)
![graph](https://user-images.githubusercontent.com/90863693/164068306-397ab65f-4c5b-4057-8a64-a7a44ac79165.png)

[extended_K115.assembly_graph.fastg.extend-embplant_pt-embplant_mt.csv](https://github.com/Kinggerm/GetOrganelle/files/8515516/extended_K115.assembly_graph.fastg.extend-embplant_pt-embplant_mt.csv)
[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/8515517/get_org.log.txt)
![graph](https://user-images.githubusercontent.com/90863693/164069139-f169e62e-8e7d-4c18-b32f-e0535f4f838a.png)

Any help would be appreciated.

Thank you

**Comment by Kinggerm** [comment]
Please update the graph with `name`, `depth`, `length` shown in Bandage.
As I can tell from the connection and the relative coverages, these two graphs are both [organelle-sufficient](https://github.com/Kinggerm/GetOrganelle/wiki/Terminology).

**Reply by leogg-github** [reply]
![graph_updated](https://user-images.githubusercontent.com/90863693/164232250-1e5d5e7e-3d4a-466e-a171-9e58838c0edf.png)
![graph_updated](https://user-images.githubusercontent.com/90863693/164232300-42e8eff8-a4b2-4a23-9490-c590befcaa0f.png)


Here are the updated graphs. The first one was organelle sufficient according to the log, but if you check the second log you will see the issues on this case. I have already tried reducing the word size but that resulted in more potential assemblies.

Thanks for your help

**Reply by leogg-github** [reply]
I just also ran using a closely related genome (using -s) and got a 6 different assemblies with 3 scaffolds each. This seems odd since I would expect a consistent assembly if I use the same species as target.

Funnily enough the second graph which corresponded to the initial run I performed with default parameters did provide a single scaffold, but my doubt remains... why is the log presenting disentangling failures?

Thanks
